### ISSUE TITLE
[Bug] Reddit Top posts filter by Duration issue for portrait mode

### ISSUE THREAD

**Issue by moizahmed97** [mozilla-mobile/fenix#7395]
## Steps to reproduce
Go to reddit.com
Click on any subreddit 
Click on the top post filter 
### Expected behavior
Should allow the user to select a duration for example "Past Day" , "Past Month" etc
### Actual behavior
The duration filter seems to get superimposed over the first post on the page and as a result cannot be selected
### Device information

* Android device: Nokia 8.1
* Fenix version: 3.0.1
![Screenshot_20191228-142252](https://user-images.githubusercontent.com/44170860/71543075-b73c7700-297f-11ea-9917-434a0897d493.png)

**Comment by sv-sdeiac** [comment]
Hello, I am not able to reproduce your issue but tapping on the "Posts" top menu it doesn't do anything.

Devices: 
- Google Pixel (Android Q);
- Samsung Galaxy S8 (Android 9);
- LG g7 fit (Android 8.1).

**Comment by kbrosnan** [comment]
@moizahmed97 would you provide a screenshot of your Firefox Settings for `Accessibility` `Font Size`. I can reproduce some font overlapping issues at 200% using a Pixel 3.

**Comment by moizahmed97** [comment]
@kbrosnan Here it is 
Setting is set to default system settings which is large 
![Screenshot_20191230-194811~2](https://user-images.githubusercontent.com/44170860/71593033-72613d80-2b43-11ea-81a9-0afc190e650a.png)
![Screenshot_20191230-194908~2](https://user-images.githubusercontent.com/44170860/71593034-73926a80-2b43-11ea-91b5-6c8108818e8c.png)

**Comment by moizahmed97** [comment]
Actually the problem is solved now, you were right its the font size that was causing issues as i had set my font setting to large, my bad

![Screenshot_20191230-195224~2](https://user-images.githubusercontent.com/44170860/71593081-a177af00-2b43-11ea-8359-750cb4860319.png)

![Screenshot_20191230-195006~2](https://user-images.githubusercontent.com/44170860/71593086-a3417280-2b43-11ea-8d8d-0f7ee43ab693.png)

Should i close the issue now?

**Comment by sv-sdeiac** [comment]
Closing this issue since the reporter confirmed the invalidity of the issue, thanks.
